appium_lib.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |s|
   s.homepage      = 'https://github.com/appium/ruby_lib' # published as appium_lib
   s.require_paths = ['lib']
 
-  s.add_runtime_dependency 'selenium-webdriver', '>= 3.0.4', '<= 3.4'
+  s.add_runtime_dependency 'selenium-webdriver', '~> 3.4.3'
```

Comment:
Because `3.4.1` has a bunch of changes and `3.4.2` has some fix and changes for them.
So, I start `3.4.3` first and go to `3.4.1` after pass all tests.